include/uv-win.h

```diff
@@ -647,3 +647,17 @@ RB_HEAD(uv_timer_tree_s, uv_timer_s);
 #ifndef X_OK
 #define X_OK 1
 #endif
+
+#if _MSC_VER >= 1900
+#undef stdin
```

Comment:
i ran vcbuild.bat in vs2015 x86 native command prompt.
yes i have vs2010, too. but i didn't run vcbuild.bat in vs2010 prompt.
and vs2015 break change like this inline function, etc.